ui/src/app/store/zone/types/actions.ts

```diff
@@ -1,11 +1,18 @@
-import type { Zone } from "./base";
-import type { ZoneMeta } from "./enum";
+import type { ZONE_PK } from "../constants";
+
+import type { Zone, ZonePK } from "./base";
 
 export type CreateParams = {
   description?: Zone["description"];
   name: Zone["name"];
 };
 
-export type UpdateParams = CreateParams & {
-  [ZoneMeta.PK]: Zone[ZoneMeta.PK];
+export type DeleteParams = {
+  [ZONE_PK]: ZonePK;
+};
+
+export type UpdateParams = {
+  [ZONE_PK]: ZonePK;
+  description?: Zone["description"];
+  name?: Zone["name"];
 };
```

Comment:
Something we should fix elsewhere - extending `UpdateParams` from `CreateParams` is slightly wrong in most cases. Some things are optional, some are not, and some aren't able to be updated, depending on the model.